NSS-Day-Cohort-49/tabloid-mvc-tabloid-mvc-cassettes
# Addposttag sx

#### Changes Made
1. Added file `PostTag` to `Model` directory.
2. Added file `PostTagRepository` and `IPostTagRepository` to `Repositories` directory.
3. Added file `PostTagViewModel` to `ViewModel` directory.

1. Modified file `Details.cshtml` in `Post` view to include `Create (Manage)` and `Delete` Post Tag functionality.
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout addposttag-sx
    ```
2. Open a new Terminal tab (?T) and navigate to the server directory.
3. Test app functionality.
> View Post Details page - bottom should provide Manage and Delete tags.
> Manage - provide list of all tags currently available and should be able to add multiple tags at once by pressing *shift + click*
> Delete - delete tags from post tag
> Post details should show all tags associated with the post
> "Back to post" link should take user back to current post details page
> Once a tag is associated with post - delete tag from tag management to see that tag is able to be deleted

4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
